chain-impl-mockchain/src/certificate/encrypted_vote_tally.rs

```diff
@@ -0,0 +1,149 @@
+use crate::transaction::{SingleAccountBindingSignature, TransactionBindingAuthData};
+use crate::vote::CommitteeId;
+use crate::{
+    certificate::{CertificateSlice, VotePlanId, VoteTallyPayload},
+    transaction::{Payload, PayloadAuthData, PayloadData, PayloadSlice},
+    vote::{PayloadType, TryFromIntError},
+};
+use chain_core::{
+    mempack::{ReadBuf, ReadError, Readable},
+    property,
+};
+use chain_crypto::Verification;
+use typed_bytes::{ByteArray, ByteBuilder};
+
+#[derive(Debug, Clone)]
+pub struct EncryptedVoteTallyProof {
+    pub id: CommitteeId,
+    pub signature: SingleAccountBindingSignature,
+}
+
+#[derive(Debug, Eq, PartialEq, Hash, Clone)]
+pub struct EncryptedVoteTally {
+    id: VotePlanId,
+    payload: VoteTallyPayload,
```

Comment:
do we really need this here? isn't it a given that the vote should be private? (`VoteTallyPayload` doesn't seem to have any more information).

Reply:
No we don't.